web/cobrands/bexley/waste.js

```diff
@@ -3,5 +3,15 @@ window.garden_waste_first_bin_discount_applies = function() {
 };
 
 $(function() {
+    // If there's an error when submitting the subscribe form,
+    // the first bin discount won't be displayed even when it should apply.
+    // This is due to how the form is set-up on the server.
+    // We work around this by just checking if the discount should apply
+    // when the page loads and running the cost calculation function if so.
+    window.onload = function() {
+        if (window.garden_waste_first_bin_discount_applies()) {
+            window.garden_waste_bin_cost_new();
+        }
+    };
```

Comment:
In theory we could fix this server side by pushing 'payment_method' into the data the cobrand method 'first_bin_discount_applies' gets in the relevant location, but it's not pretty, and this is probably fine.